### ISSUE TITLE
[preprocess-inline-svg] Build-time error is thrown when optional config object is omitted in call to `inlineSvg()`

### ISSUE THREAD

**Issue by davidmreed** [vnphanquang/svelte-put#191]
The documentation shares the following example for using `preprocess-inline-svg` as a SvelteKit preprocessor:

```
import inlineSvg from '@svelte-put/preprocess-inline-svg';

/** @type {import('@sveltejs/kit').Config} */
const config = {
  preprocess: [
    // input to `inlineSvg` can be either a single object or an array of such
    inlineSvg([
      {
        directories: 'src/assets/icons',
        attributes: {
          class: 'icon',
          width: '20',
          height: '20',
        },
      },
      {
        directories: 'src/assets/pictograms',
      },
    ]),
    // other preprocessors
  ],
};
export default config;
```

However, the build actually fails with the following error:

```
error during build:
TypeError: Cannot read properties of undefined (reading 'inlineSrcAttributeName')
    at resolveInlineSvgConfig (file:///home/dmr/Projects/x/node_modules/@svelte-put/preprocess-inline-svg/dist/index.js:89:42)
    at inlineSvg (file:///home/dmr/Projects/x/node_modules/@svelte-put/preprocess-inline-svg/dist/index.js:182:19)
    at file:///home/dmr/Projects/sharethepayband/svelte.config.js?ts=1682704087611:10:3
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
    at async load_config (file:///home/dmr/Projects/x/node_modules/@sveltejs/kit/src/core/config/index.js:70:17)
    at async sveltekit (file:///home/dmr/Projects/x/node_modules/@sveltejs/kit/src/exports/vite/index.js:116:24)
    at async Promise.all (index 0)
    at async asyncFlatten (file:///home/dmr/Projects/x/node_modules/vite/dist/node/chunks/dep-3007b26d.js:12365:16)
```

unless the optional second argument to `inlineSvg()` is populated:

```
		inlineSvg(
			{
				directories: 'static/svgs',
				attributes: {}
			},
			{
			}
		),
```

The typing in the package also suggested to me that the second parameter is meant to be optional.

**Comment by vnphanquang** [comment]
Hello @davidmreed thank you for reporting. I'll take a look soon

**Comment by vnphanquang** [comment]
Should be fixed in [@svelte-put/preprocess-inline-svg@2.1.1](https://github.com/vnphanquang/svelte-put/releases/tag/%40svelte-put/preprocess-inline-svg%402.1.1). 

Closing for now, please let me know if you're still having problems. And thanks again for reporting 🙏🏼
